src/features/projects/projectsService.js

```diff
@@ -0,0 +1,26 @@
+import axios from "axios";
+const ALL_PROJECTS_URL = "http://localhost:8080/api/projects/"
+
+//get all projects
+const getProjects = async () => {
+	const PROJECTS_URL = "http://localhost:8080/api/projects/"
```

Comment:
is there any difference between PROJECTS_URL and ALL_PROJECTS_URL
ALL_PROJECTS_URL is not used
create all constants fro url outside

Reply:
ALL_PROJECTS_URL was left here in error, I've made the changes requested